packages/algorand-js/src/errors/errors-list.ts

```diff
@@ -22,14 +22,14 @@ const tealErrors = {
     message: "Length of stack is less than min length required for current op",
     title: "Length of stack is less than min length required for current op",
     description: `You are trying to perform an operation on stack where the stack does not
-have sufficient length.`
+    have sufficient length.`
```

Comment:
weird. `linter` didn't allowed me add tabs. Gave something like `no extra tabs`. Did you change some rule?

Reply:
No, Please check your linter. I ran yarn lint:fix